### PR TITLE
chore: Add a11y checks for home and about page

### PR DESCRIPTION
**Pull request by AliceR**
This is the basic structure to check each page for accessibility violations. Starting with home and about page for now.
There are a couple of issues listed in the violations.

We can tackle them, but also we can look into the rules these checks are based on.

> By default, axe checks against a wide variety of accessibility rules. Some of these rules correspond to specific success criteria from the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/), and others are "best practice" rules that are not specifically required by any WCAG criterion.
> 
>You can constrain an accessibility scan to only run those rules which are "tagged" as corresponding to specific WCAG success criteria [...].

https://playwright.dev/docs/accessibility-testing#scanning-for-wcag-violations

---
Note: the checks fail right now! Since I am using my previous branch as a base, and not main or develop, the playwright checks don't run on CI for this branch. We do not want to block CI, hence need to be careful merging this as is! We need to fix the pages first.

### CODE REVIEW

**Comment by AliceR** [comment]
To keep our accessibility testing focused and manageable, we could configure axe to test against a specific standard or subset of rules, rather than enabling all available checks. Axe supports a tagging system that lets us target specific standards like wcag2a, wcag2aa, best-practice, etc. (full list here: https://www.deque.com/axe/core-documentation/api-documentation/#axe-core-tags).

A good starting point might be to focus on wcag2a and wcag2aa, which align with common legal and industry requirements. If we start seeing a high volume of violations, it may be helpful to narrow our scope further — either by prioritizing the most critical categories (e.g., keyboard navigation, contrast, ARIA) or defining a custom minimum baseline for passing.

This way, we stay pragmatic: catching real accessibility blockers without being overwhelmed by lower-priority or edge-case issues.

**Comment by AliceR** [comment]
Should we limit the checks to just wcag2a and wcag2aa for now? That might help us focus on catching real accessibility blockers without getting overwhelmed by lower-priority or edge-case issues. We can always expand the scope later if needed.

File: test/integration/a11y.spec.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { test, expect } from '@playwright/test';
+import AxeBuilder from '@axe-core/playwright';
+
+test.describe('homepage', () => {
```
Comment (AliceR) [inline test/integration/a11y.spec.ts:4]:
```suggestion
test.describe('Homepage', () => {
```

**Review by dzole0311 — APPROVED** [review]
Changes look good. Before merging, as discussed, we should attempt fix the few accessibility issues in next-earth-gov so the build doesn't fail (if they aren't too big of a change). If the issues come from the veda-ui components, we can ignore them until we implement something similar on the veda-ui side as well.
